cli/common/pipeline.go

```diff
@@ -18,6 +18,11 @@ func DetectPipelineConfig() (multiplies bool, config string, _ error) {
 		return true, config, nil
 	}
 
+	config = ".woodpecker.yaml"
```

Comment:
we should use mentioned const asap implemented ...

Reply:
also moved to shared/constant